mailboxes/lang/de

```diff
@@ -180,20 +181,20 @@ view_hammove=Berichte Spam und Eingang
 view_razordel=Spam-Report und Löschen
 view_dstatus=Status des gescheiterten Versands
 view_dstatusok=Erfolgreicher Übertragungsstatus
-view_final-recipient=Entgültiger Empfänger
+view_final-recipient=Ungültiger Empfänger
 view_diagnostic-code=Grund des Fehlers
 view_remote-mta=Entfernter Mail-Server
 view_reporting-mta=Berichtender Mail-Server
-view_astext=Zeige Mail als Text an (empfohlen)
-view_ashtml=Zeige Mail als HTML an (vorsicht!)
+view_astext=Mail als Text anzeigen (empfohlen)
+view_ashtml=Mail als HTML anzeigen (Vorsicht!)
```

Comment:
No need for `(Vorsicht!)` and `(empfohlen)`. These are obsolete strings based on [source](https://github.com/webmin/webmin/blob/master/mailboxes/lang/en) file.